db/db.go

```diff
@@ -176,6 +176,15 @@ func setupDb(tx *sql.Tx) error {
 
 		`ALTER TABLE sessions DROP COLUMN IF EXISTS active`,
 		`CREATE TABLE IF NOT EXISTS activeClientSessions (uuid BINARY(16) NOT NULL PRIMARY KEY, clientSessionId VARCHAR(32) NOT NULL, FOREIGN KEY (uuid) REFERENCES accounts (uuid) ON DELETE CASCADE ON UPDATE CASCADE)`,
+	
+		`CREATE TABLE IF NOT EXISTS friends (
```

Comment:
why not `uuid` as a key? It is already indexed and it is better in terms of design to refer primary keys for foreign keys

Reply:
Yeah you're right, I did this at the very start to avoid having to convert from friend username to its uuid, but it doesn't matter since it will be rarely called and even if it was called often, I can remove a lot of `FetchUsernameFromUUID` this way. Idk why I did not changed it sooner, it's better indeed